packages/oauth/src/callback-options.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { assert } from 'chai';
+import { describe, it } from 'mocha';
+import sinon from 'sinon';
+import { IncomingMessage, ServerResponse } from 'http';
+
+import { CallbackOptions } from './callback-options';
+import { MissingStateError } from './errors';
+
+describe('CallbackOptions', async () => {
+  it('should have success and failure', async () => {
```

Comment:
These tests are not so great but it's better than nothing

Reply:
Indeed, only about ~62% of branches are tested in this package, so definitely room for improvement!